rocky/reports/views/aggregate_report.py

```diff
@@ -134,72 +131,6 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class SaveAggregateReportMixin(ReportPluginView):
```

Comment:
Moved to `mixins.py`

Reply:
This is aggregate specific code and not a general reporting mixin. What is the reason to moving it to the general mixins file instead of keeping it in the aggregate specific file? It doesn't seem a logical place to me.

Reply:
If you hadn't unnecessarily move this there also wouldn't have been merge conflicts here.

Reply:
I have move this code to mixins to keep the flow of the report wizard in the code more readable. As it follows a path top down.

Reply:
I can also put it at the bottom of the code.